documentation-site/components/yard/config/combobox.ts

```diff
@@ -39,6 +39,15 @@ const ComboboxConfig: TConfig = {
         into: 'value',
       },
     },
+    onFocus: {
+      value: 'nextValue => setValue(nextValue)',
+      type: PropTypes.Function,
+      description: 'Callback for when input focus changes.',
+      propHook: {
+        what: 'nextValue',
+        into: 'value',
+      },
```

Comment:
Can you please remove this? To reproduce the bug, go to the interactive example [here](https://baseweb-git-jonsadka-focus-combobox.uber-ui-platform.vercel.app/components/combobox) select an option, blur, then focus and you will see `[object Object]` in the input field.

```suggestion
```

Reply:
Looks like this was missed in an earlier PR. Can you please this same `propHook` value from L29 above while looking at the file? Sorry about the confusion

Reply:
Done